pangeo_forge_runner/commands/bake.py

```diff
@@ -55,6 +55,28 @@ class Bake(BaseCommand):
         """,
     )
 
+    recipe_id = Unicode(
+        None,
+        allow_none=True,
+        config=True,
+        help="""
+        Optionally pass this value to run only this recipe_id from the feedstock.
+
+        If empty, all recipes from the feedstock will be run.
+        """,
+    )
+
+    job_name = Unicode(
+        None,
+        allow_none=True,
+        config=True,
+        help="""
+        Optionally pass a custom job name for the job run.
+
+        If empty, a name will be generated for the job below.
```

Comment:
```suggestion
        If empty, a unique name will be generated for the job.
```

This documentation is also read by end users, so 'below' doesn't make sense outside of the code.